src/utils/currencies.js

```diff
@@ -40,7 +40,8 @@ function getFixedTokenAmount(amount, decimals) {
   // We can lose precision as there will never be more than MAX_DECIMALS_UNTIL_ZERO_AMOUNT significant digits
   const balanceWithDecimals = Number(amount) / Math.pow(10, decimals);
 
-  return balanceWithDecimals.toFixed(MAX_TOKEN_DECIMALS);
+  // Double conversion to remove zeros and return a string
+  return Number(balanceWithDecimals.toFixed(MAX_TOKEN_DECIMALS)).toString();
```

Comment:
It's not a big deal since the merge will probably be automatic, but yeah, we better implement things in one place so the review discussion is not duplicated.